queries/cow_rewards.sql

```diff
@@ -17,13 +17,13 @@ with trade_hashes as (SELECT solver,
                           on settlement.tx_hash = solver_competitions.tx_hash
                       where block_number between {{start_block}} and {{end_block}})
 
-select concat('0x', encode(solver, 'hex'))          as receiver,
-       -- Used to compare with Dune (cross referencing that all trades are accounted for in the orderbook)
-       count(*) as num_trades,
-       (sum(coalesce(reward, 0)) * pow(10, 18))::numeric::text   as amount,
-       '0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB' as token_address
+select concat('0x', encode(solver, 'hex'))                as solver,
+       concat('0x', encode(tx_hash, 'hex'))               as tx_hash,
+--        concat('0x', encode(trade_hashes.order_uid, 'hex')) as order_uid,
```

Comment:
Nit: maybe remove?

Reply:
Actually, if we are going to want to test this on a per order level, then we may want to un-comment here. However the proposed tests are per batch so I will remove for now.